Formula/terraform.rb

```diff
@@ -52,21 +54,27 @@ def install
       buildpath.install "stringer"
     end
 
+    cd "src/github.com/kisielk/errcheck" do
+      system "go", "build"
+      buildpath.install "errcheck"
+    end
+
     cd terrapath do
       # v0.6.12 - source contains tests which fail if these environment variables are set locally.
       ENV.delete "AWS_ACCESS_KEY"
       ENV.delete "AWS_SECRET_KEY"
 
-      terraform_files = `go list ./...`.lines.map { |f| f.strip unless f.include? "/vendor/" }.compact
-      system "go", "test", *terraform_files
+      # Runs format check and test suite via makefile
+      ENV.deparallelize { system "make", "test", "vet" }
 
-      mkdir "bin"
+      # Generate release binary
       arch = MacOS.prefer_64_bit? ? "amd64" : "386"
-      system "gox",
-        "-arch", arch,
-        "-os", "darwin",
-        "-output", "bin/terraform-{{.Dir}}", *terraform_files
-      bin.install "bin/terraform-terraform" => "terraform"
+      ENV["XC_OS"] = "darwin"
+      ENV["XC_ARCH"] = arch
+      system "make", "bin"
+
+      # Install release binary
+      bin.install "pkg/darwin_#{arch}/terraform"
```

Comment:
The build places the file (in addition to the above place) in the `bin/` relative directory, as well as the `$GOPATH/bin/` directory. So while they're in the right places in the tmp files, homebrew doesn't pick up on them to actually move them over to the Cellar without the install command.

Any leads as to why?